apps/charge-apps-service/src/common/config/configuration.ts

```diff
@@ -1,4 +1,6 @@
 export default () => ({
+  CHARGE_BASE_URL: process.env.CHARGE_BASE_URL,
```

Comment:
should be cammel cased, please see how we define the other variables.

Reply:
It was done in order to match this getter in charge-api.service: 
![Screenshot 2023-11-13 at 11 20 23](https://github.com/fuseio/fusebox-backend/assets/68244817/c92f49f7-17bc-4e79-9491-0c1df335eb3b)
For some reason we were unable to get it from .env